posts/abcs-of-dqm-balance

```diff
@@ -0,0 +1,80 @@
+---
+authors:
+  - Pooja Krishnan
+tags:
+  - Data
+date: 2020-06-15T14:50:55.000Z
+title: 'The ABCs of DQM: Balance'
+image: https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2020/07/abcs-dqm-balance-logo.png
+---
+
+This post is a part of a series of posts on Data Quality Management. The focus of this series is on strategies for acquiring data that can be used to ensure the quality of data. The five aspects of Data Quality Management are:
+
+1. Accuracy
+2. Consistency
+3. Completeness
+4. Integrity
+5. Timeliness
+
+In this article, we will expand on the definition of the ABC Framework as [mentioned in this article](https://blog.ippon.tech/abcs-of-dqm-audit/), written by my colleague [Daniel Ferguson](https://www.linkedin.com/in/daniel-ferguson-985b7048/). There he introduces the ABC Framework of Data Quality Management, mainly focusing on how the Audit process lays the groundwork for high-quality data.
+
+In this blog, we will look at the B in the ABC framework, Balance, and how the Balance process on an ETL pipeline can confirm your data's accuracy and consistency. We'll also examine how the metadata collected as a part of the Audit process can be used to Balance your data.
+
+# How do you Balance your Data?
+
+Balance, to put it simply, is how you can confirm if your ETL processes are operating with the correct data. Consider a situation where you have an ETL process that reads data from a source table's partition (from another system) and writes that partition to a target table within your system. How can you tell that all the data from your source partition made it to your target partition? If you already have an audit process that captures metadata, you can use those metrics to balance your data. Consider the below sample data model for an Audit table created using [SQLDBM](https://app.sqldbm.com/)
+
+![Audit Schema Example](https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2020/07/abcs-dqm-balance-1.png)
+
+With this model, we know how each job ran, and where those job's records came from and were stored. This audit information will form the basis for our balance process.
```

Comment:
With this model, we know how each job ran, and where each job's records came from and were stored.